projects/an-qrcode/package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "an-qrcode",
-  "version": "1.0.5",
+  "version": "1.1.1",
   "author": {
     "name": "Naim Malek <naimmalek786@gmail.com>",
     "url": "https://naimmalek.github.io"
   },
-  "description": "Angular6/7/8/9/10 QRCode generator",
+  "description": "Angular QRCode generator",
   "peerDependencies": {
-    "@angular/common": ">= 6.0.0 < 9.0.6",
-    "@angular/core": ">= 6.0.0 < 9.0.6"
+    "@angular/common": ">= 6.0.0 < 18.0.0",
```

Comment:
I'm unsure how things work around here, but it seems like if you insist on having a _maximum_ version you'll forever play catchup, needing to publish a new release every time a new major Angular version comes out 🤔 

Perhaps you can remove the _maximum_ version 🤷 - I wish I knew better how it all worked 🤝

Reply:
Yes you are correct. Please remove max version from here

Reply:
✅ removed 🪓 😁